screener/models.py

```diff
@@ -388,6 +388,33 @@ def is_married(self):
     def has_disability(self):
         return self.disabled or self.visually_impaired or self.long_term_disability
 
+    def is_head(self):
+        return self.relationship == 'headOfHousehold'
+
+    def is_spouse(self):
+        return self.screen.relationship_map()[self.screen.get_head().id] == self.id
+
+    def is_dependent(self):
+        is_tax_unit_spouse = self.is_spouse()
+        is_tax_unit_head = self.is_head()
+        is_tax_unit_dependent = (
```

Comment:
More of a business logic question here. I saw on the [IRS website](https://apps.irs.gov/app/vita/content/globalmedia/4491_dependency_exemptions.pdf) that dependents include "a child, stepchild, brother, sister, or parent." Our definition of `is_depent` doesn't seem to include parents, siblings, step child, .etc, are we purposely leave them out here? Thanks.

Reply:
People over 18 require special cases to be qualified as a dependent, such as being a student, or being disabled. We do account for these edge cases on lines 403 and 404.